src/control/cmd/daos/flags_test.go

```diff
@@ -562,3 +562,55 @@ func TestFlags_FsCheckFlag(t *testing.T) {
 		})
 	}
 }
+
+func TestFlags_ModeBitsFlag(t *testing.T) {
+	for name, tc := range map[string]struct {
+		arg     string
+		expFlag *ModeBitsFlag
+		expErr  error
+	}{
+		"unset": {
+			expErr: errors.New("empty file mode flag"),
+		},
+		"not an integer": {
+			arg:    "foo",
+			expErr: errors.New("invalid mode: \"foo\""),
+		},
+		"invalid octal": {
+			arg:    "0999",
+			expErr: errors.New("invalid mode: \"0999\""),
+		},
+		"negative value": {
+			arg:    "-0755",
+			expErr: errors.New("invalid mode: \"-0755\""),
+		},
+		"valid": {
+			arg: "0755",
+			expFlag: &ModeBitsFlag{
+				Set:  true,
+				Mode: 0755,
+			},
+		},
+		"valid with leading zero": {
+			arg: "755",
```

Comment:
nit - seems in this case it is with*out* the leading zero

Reply:
done